packages/tempus/src/Tempus_StepperDIRK_decl.hpp

```diff
@@ -175,26 +179,26 @@ class StepperDIRK : virtual public Tempus::StepperImplicit<Scalar>,
 
     /// Take the specified timestep, dt, and return true if successful.
     virtual void takeStep(
-      const Teuchos::RCP<SolutionHistory<Scalar> >& solutionHistory);
+      const Teuchos::RCP<SolutionHistory<Scalar> >& solutionHistory) override;
 
     /// Get a default (initial) StepperState
-    virtual Teuchos::RCP<Tempus::StepperState<Scalar> >getDefaultStepperState();
+    virtual Teuchos::RCP<Tempus::StepperState<Scalar> >getDefaultStepperState() override;
 
-    virtual bool isExplicit() const
+    virtual bool isExplicit() const override
     {
       const int numStages = this->tableau_->numStages();
       Teuchos::SerialDenseMatrix<int,Scalar> A = this->tableau_->A();
       bool isExplicit = false;
       for (int i=0; i<numStages; ++i) if (A(i,i) == 0.0) isExplicit = true;
```

Comment:
This only tests that the diagonal is zero. But would fail for `A(0,1) != 0`. This is a bug

Reply:
Yeah, I guess it only tests the diagonal, but this is the DIRK class.  It would fail the isDIRK test.  Adding isDIRK test to catch this.